packages/console/src/components/Table/types.ts

```diff
@@ -0,0 +1,20 @@
+import type { Key, ReactNode } from 'react';
+import type { FieldPath, FieldPathValue, FieldValues } from 'react-hook-form';
+
+export type Column<
+  TFieldValues extends FieldValues = FieldValues,
+  TName extends FieldPath<TFieldValues> = FieldPath<TFieldValues>
+> = {
+  title: ReactNode;
+  dataIndex: TName;
+  render: (value: FieldPathValue<TFieldValues, TName>, row: TFieldValues) => ReactNode;
+  colSpan: number;
```

Comment:
colSpan should be optional, and having a default value of 1

Reply:
updated